modules/apps/commerce/commerce-theme-minium/commerce-theme-minium/src/css/minium/overrides/_data-set-display.scss

```diff
@@ -86,4 +87,49 @@
 	.table-style-stacked {
 		border: none;
 	}
+
+	/* Custom Minium table cell customization */
+
+	div[data-id='com_liferay_commerce_order_content_web_internal_portlet_CommerceOrderContentPortlet-placedOrderItems'] {
+		.dnd-td,
+		.dnd-th {
+			&.cell-sku {
+				min-width: 100px;
+			}
+
+			&.cell-shippedQuantity {
+				max-width: 150px;
+			}
+
+			&.cell-promoPrice {
+				max-width: 132px;
+			}
+
+			&.cell-name {
+				min-width: 150px;
+			}
+		}
+	}
+
+	div[data-id$='Orders'],
+	div[data-id$='OrderItems'] {
```

Comment:
I like the idea of coalescing same rules for similar tables. This is doable with classes too, though to support regex we need something like `div[class$='OrderItems']`, not exactly the same as a dedicated attribute but I think we're fine